### PR TITLE
Allowed manual input of reaction and metabolite database in rBioNet

### PR DESCRIPTION
**Pull request by almut-heinken**
Enables the manual input of reaction and metabolite database instead of through GUI.

**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [X] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [ ] with [X] to check the box)*

### CODE REVIEW

File: src/reconstruction/rBioNet/data2model.m [hunk 1]
```diff
@@ -9,10 +9,18 @@
 %    data:          data from `rbionet`
 %    description:   model description
 %
+% OPTIONAL INPUT
```
Comment (laurentheirendt) [inline src/reconstruction/rBioNet/data2model.m]: This needs proper documentation formatting.

File: src/reconstruction/rBioNet/data2model.m [hunk 2]
```diff
@@ -138,8 +146,14 @@
 % max_msgbox = 0;
 %
 
-reactions   = rBioNetSaveLoad('load','rxn');
-metabolites = rBioNetSaveLoad('load','met');
+% if reaction and metabolite database is input manually instead of through rBioNet GUI
+if nargin>2
```
Comment (laurentheirendt) [inline src/reconstruction/rBioNet/data2model.m]: May you use `if isfield(...)`?

Comment (almut-heinken) [reply src/reconstruction/rBioNet/data2model.m]: Is that really such a good idea? Then there is a "not enough input arguments" error when database is not entered and every script using this function would need to be rewritten.

Comment (laurentheirendt) [reply src/reconstruction/rBioNet/data2model.m]: Not sure I understand ...

Comment (almut-heinken) [reply src/reconstruction/rBioNet/data2model.m]: Never mind,...that should be isstruct, not isfield.

Comment (laurentheirendt) [reply src/reconstruction/rBioNet/data2model.m]: Sorry, I wanted to say: `if ~exist('database', 'var')` :smile:

File: src/reconstruction/rBioNet/data2model.m [hunk 3]
```diff
@@ -147,7 +148,7 @@
 %
 
 % if reaction and metabolite database is input manually instead of through rBioNet GUI
-if nargin>2
+if isfield(database,'reactions') && isfield(database,'metabolites')
```
Comment (tpfau) [inline src/reconstruction/rBioNet/data2model.m]: You will still need to test, whether database actually exists
i.e.
```
if exist('database','var') && isfield(database,'reactions') && isfield(database,'metabolites')
...
```

Comment (almut-heinken) [reply src/reconstruction/rBioNet/data2model.m]: Ok, fine.

**Comment by laurentheirendt** [comment]
Build on ARTENOLIS is failing because of an unrelated test.

**Comment by laurentheirendt** [comment]
Thanks @almut-heinken 👍

**Pull request merged**